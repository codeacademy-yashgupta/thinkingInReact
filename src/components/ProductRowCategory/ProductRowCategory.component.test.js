import React from 'react';
import ProductRowCategory from './ProductRowCategory.component';
import renderer from 'react-test-renderer';

describe('ProductRowCategory', () => {
  it('should render product row category', () => {
    const tree = renderer.create(<ProductRowCategory/>).toJSON();
    expect(tree).toMatchSnapshot();
  })
});