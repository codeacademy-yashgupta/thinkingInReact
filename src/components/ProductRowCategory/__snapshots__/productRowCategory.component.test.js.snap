// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductRowCategory should render product row category 1`] = `
<tr>
  <th
    colSpan="2"
  />
</tr>
`;
